source/dotnet/Library/AdaptiveCards.Rendering.Wpf/RenderedAdaptiveCard.cs

```diff
@@ -16,15 +18,35 @@ public RenderedAdaptiveCard(FrameworkElement frameworkElement, AdaptiveCard orig
         /// </summary>
         public FrameworkElement FrameworkElement { get; }
 
+        internal Dictionary<string, Func<object>> InputBindings { get; set; } = new Dictionary<string, Func<object>>();
+
 
         /// <summary>
         /// Event handler for when user invokes an action.
         /// </summary>
-        public event EventHandler<AdaptiveActionEventArgs> OnAction;
+        public event TypedEventHandler<RenderedAdaptiveCard, AdaptiveActionEventArgs> OnAction;
 
         internal void InvokeOnAction(AdaptiveActionEventArgs args)
         {
             OnAction?.Invoke(this, args);
         }
+
+        /// <summary>
+        /// Gather the current values from any inputs on the card
+        /// </summary>
+        /// <returns></returns>
+        public virtual JObject GetInputData()
```

Comment:
>GetInputData [](start = 31, length = 12)

This is diverged from what the UWP API is. I believe the UWP API has a UserInputs property, and then you can call AsJson() or AsValueSet().